### ISSUE TITLE
Log files not flushing?

### ISSUE THREAD

**Issue by BunnyMerz** [luiseduve/exciteometer#18]
I am attempting to save the logs for the excite-device output, along with the coordinates of a random object on the scene. I see Unity creating the `LogFiles_ExciteOMeter/logDebug`, but all it saves is the initial moment of the app, where it has no incoming data:

```txt
datetime,Event
[...],New App Execution
[...],ExciteOMeter >> No incoming data RRInterval was found to calculate feature SDNN
datetime,Event
[...],New App Execution
```

I am able to receive data from the polar and show on screen using the online analysis. Why is it not logging the rest of the information? Does the package constantly update the file, or do I need to gracefully stop the game to properly save the current data stream? Besides, shouldn't there be some `.csv` files on that same directory?

Since I am talking about this subject, here goes a question: Would the Simulation prefab trigger the logger to save the simulated data as well? Thanks for any help!

**Comment by luisqtr** [comment]
Hi, the simulation prefab would work as if the real sensor was collected. But note that the `logDebug` doesn't have a real usage in the project, the main log files with the physiological data should appear at the same level than the Assets folder in Unity, but it will only be created once you press the `Start/Stop session` button in the UI, the log does not start automatically when clicking `Play` in the Editor.

**Comment by BunnyMerz** [comment]
Hi! Thanks for the response, and yeah that makes sense. Also, I have previously attempted to click in any of the UI elements, but none of them result in any action/trigger.

I attempted to call `StartStopRecording` within `LogMessageUI.cs > Update()` to bypass the button press. I attached it to a random input and it worked! Not sure why the button isn't working, but I will be activating it programmatically anyways, so this ought to do it.

Thanks for the help!
